website/blog/2025/09-15/index.md

```diff
@@ -0,0 +1,404 @@
+---
+slug: crawlee-for-python-v1
+title: Crawlee for Python v1
+description: Announcing the Crawlee for Python v1.0.0 release.
+authors: [VladaD]
+---
+
+We are happy to announce that Crawlee for Python v1.0.0 is finally here. After months of development, polishing, and feedback from the community, the library is leaving beta and entering a production/stable development status. This reflects our confidence in the API and the internal architecture. And from now on Crawlee for Python will strictly follow [semantic versioning](https://semver.org/), which means you can rely on it as a stable foundation for your crawling and scraping projects, knowing that breaking changes will only occur in major releases.
+
+![Crawlee for Python v1.0.0](./img/crawlee_v100.webp)
```

Comment:
the image says v0.6.0 but i guess you are aware of that

Reply:
yeah, I already requested the update from the design team